assets/components/tinymcerte/js/vendor/tinymce/skins/modx/skin.min.css

```diff
@@ -324,6 +343,19 @@ i.mce-i-checkbox {
 	color: #aa0000;
 }
 
+.mce-menu {
+	-moz-box-shadow: none;
+	-webkit-box-shadow: none;
+	box-shadow: none;
+	margin: 2px 0 0;
+	max-height: 400px;
+	min-width: 160px;
+}
+
+.mce-menu .mce-menu-item-sep, .mce-menu-item-sep:hover {
```

Comment:
Maybe the important rule could be removed with a better specifity by adding .mce-menu to the second rule.

`.mce-menu .mce-menu-item-sep, .mce-menu .mce-menu-item-sep:hover {`

Reply:
It doesn't work without `!important`. 
p.s. The response form got buggy and my previous comment disappeared.